canary/apps/react/next/package.json

```diff
@@ -24,7 +25,7 @@
   "size-limit": [
     {
       "path": ".next/static/**/*.js",
-      "limit": "350 kB"
+      "limit": "530 kB"
```

Comment:
How was this number determined?

Reply:
MJ ran the build on a PoC PR and saw that the value increased - https://github.com/aws-amplify/amplify-ui/pull/3693

@0618 do you have the link to the failing run where we saw the new value?

Reply:
Sure, that was this build

https://github.com/aws-amplify/amplify-ui/actions/runs/4661483627/jobs/8250800510

![image](https://user-images.githubusercontent.com/11983489/231295218-71b99122-b911-44e4-a58a-f3007a369f8c.png)